js/app.js

```diff
@@ -0,0 +1,234 @@
+    { "title" : "Diamonds Are Forever", "year" : 1971, "actor" : "Sean Connery", "gross" : "$648,514,469" },
+    { "title" : "Quantum of Solace", "year" : 2008, "actor" : "Daniel Craig", "gross" : "$622,246,378" },
+    { "title" : "From Russia with Love", "year" : 1963, "actor" : "Sean Connery", "gross" : "$576,277,964" },
+    { "title" : "Die Another Day", "year" : 2002, "actor" : "Pierce Brosnan", "gross" : "$543,639,638" },
+    { "title" : "Goldeneye", "year" : 1995, "actor" : "Pierce Brosnan", "gross" : "$529,548,711" },
+    { "title" : "On Her Majesty's Secret Service", "year" : 1969, "actor" : "George Lazenby", "gross" : "$505,899,782" },
+    { "title" : "The World is Not Enough", "year" : 1999, "actor" : "Pierce Brosnan", "gross" : "$491,617,153" },
+    { "title" : "For Your Eyes Only", "year" : 1981, "actor" : "Roger Moore", "gross" : "$486,468,881" },
+    { "title" : "Tomorrow Never Dies", "year" : 1997, "actor" : "Pierce Brosnan", "gross" : "$478,946,402" },
+    { "title" : "The Man with the Golden Gun", "year" : 1974, "actor" : "Roger Moore", "gross" : "$448,249,281" },
+    { "title" : "Dr. No", "year" : 1962, "actor" : "Sean Connery", "gross" : "$440,759,072" },
+    { "title" : "Octopussy", "year" : 1983, "actor" : "Roger Moore", "gross" : "$426,244,352" },
+    { "title" : "The Living Daylights", "year" : 1987, "actor" : "Timothy Dalton", "gross" : "$381,088,866" },
+    { "title" : "A View to a Kill", "year" : 1985, "actor" : "Roger Moore", "gross" : "$321,172,633" },
+    { "title" : "License to Kill", "year" : 1989, "actor" : "Timothy Dalton", "gross" : "$285,157,191" }
+];
+// // add titles from array object to empty array
+// let array1 = [1,2,3,4]
+// array1.push(5)
+
+// // create an empty array called bond titles
+// let bondTitles = [];
+// // go through each item of array
+// for(i=0; i < bondFilms.length; i++){
+//     // add/push this title to  array
+//     bondTitles.push(bondFilms[i].title)
+// }
+// console.log(bondTitles);
+//
+// // Add odd number of years to empty array from an array object
+// let oddBonds = [];
+// for(i=0; i < bondFilms.length; i++){
+//     if([bondFilms[i].year] % 2 === 1){
+//
+//         oddBonds.push(bondFilms[i].year)
+//     }
+// }
+// console.log(oddBonds);
+
+// console.log(bondFilms[0].gross);
+
+/*
+// it keeps a running total
+let total = 0;
+
+// iterate over bondFilms
+for (i=0; i < bondFilms.length; i++){
+    // get gross for this film
+    let money = bondFilms[i].gross;
+
+    //take the first charater off string
+    let gross = money.substr(1).replace(/,\s?/g, ""); // copied .replace(/,\s?/g, "") it somehow removed all commas
+
+    // turn gross for this film into a number
+    let number = parseInt(gross);
+    // console.log(number)
+
+
+    // add number to total so far
+    total = number + total;
+
+}
+console.log(total);
+// console.log(bondFilms[i].gross);
+*/
+/***************DIGGING DEEPER ****************************/
```

Comment:
**1/7** on this section -- +1 for just showing some kind of attempt to think through it